gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/activity/CommitActivity.java

```diff
@@ -32,5 +34,5 @@ public interface CommitActivity {
    * @return number of workunits committed
    */
   @ActivityMethod
-  int commit(WUProcessingSpec workSpec);
+  CommitGobblinStats commit(WUProcessingSpec workSpec);
```

Comment:
to me, "gobblin commit stats" sounds better than ""commit gobblin stats", since of all the gobblin stats, these are the commit ones, rather than of all the commit stats these are commit ones.

that said, as all this is within our gobblin impl, this could probably be simply `CommitStats`